apps/cugetreg-api/src/user/user.graphql

```diff
@@ -23,18 +22,36 @@ input CourseCartItemInput {
   academicYear: String!
   courseNo: String!
   semester: String!
+
+  """
+  The section no. that user selected for this course.
+  """
   selectedSectionNo: String!
   isHidden: Boolean!
   color: String
 }
 
 type Query {
+  """
+  Returns current user.
+  Requires user authentication.
+  """
   me: User
+
+  """
+  Returns current user's course cart.
+
+  Requires user authentication.
+  """
   courseCart: [CourseCartItem!]
-  calendarId: String
+
+  calendarId: String @deprecated(reason: "Will be redesigned.")
```

Comment:
Is it unused?

Reply:
Im not sure if frontend currently accesses it or not. If not, then we can just remove.